codefori/vscode-db2i
# Go to Definition support

Enables Go to Definition and Peek Definition, which under the hood uses `GENERATE_SQL`.

~We are using our legacy formatter for the time being. Formatted issues won't be solved here.~

* [x] Bring in changes from #326 
* [x] Remove use of formatter since we get formatted results

## To test

1. Write an SQL statement: `select * from sndbx3.sales`
2. Right click on `sndbx3.sales` and use either: _Go to Definition_ or _Peek_ -> _Definition_
3. A slimmed down statement to create the referenced object should appear
